html/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>GWEALTH VISUALS PORTFOLIO</title>
+    </head>
+    <body>
+        <header>
+            <div>
+                <img src="https://media.licdn.com/dms/image/C4D0BAQFYonLvnmZNJw/company-logo_200_200/0/1676624546613?e=1690416000&v=beta&t=DD0FGcJL_M6Gj0r_DfxmynuFdSewPiSTD4ug4MGaMlo" alt="my logo"/>
+                <h1>Hello</h1>
+                <h2>Welcome to Gwealth Visiuals</h2>
+            </div>
+        </header>
+        <nav>
+            <ul>
+                <li><a href="#">Portfolio</a></li>
+                <li><a href="#">Blog</a></li>
+                <li><a href="#">Brandings</a></li>
+                <li><a href="#">Showreel</a></li>
+                <li><a href="#">Testimonial</a></li>
+                <li><a href="#">Contact Us</a></li>
+            </ul>
+        </nav>
+        <section>
+            <h3>"Motto: Zenith of Creativity"</h3>
```

Comment:
HI pal, the instruction says, "Within the <section> element, add an <h1> element with a main heading ".

Reply:
Thank you so much sir Melumm, I have corrected the element , Please check my code @Melumm